src/Http/Role/UpdateRoleRequestHandler.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Json;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Role\Commands\RenameRole;
+use CultuurNet\UDB3\Role\MissingContentTypeException;
+use CultuurNet\UDB3\Role\UnknownContentTypeException;
+use Fig\Http\Message\StatusCodeInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Slim\Psr7\Response;
+
+class UpdateRoleRequestHandler implements RequestHandlerInterface
+{
+    private CommandBus $commandBus;
+
+    public function __construct(CommandBus $commandBus)
+    {
+        $this->commandBus = $commandBus;
+    }
+
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        $this->ensureContentTypeIsProvided($request);
+
+        $routeParameters = new RouteParameters($request);
+        $roleId = new UUID($routeParameters->get('id'));
+
+        $body = Json::decodeAssociatively($request->getBody()->getContents());
+
+        $this->commandBus->dispatch(new RenameRole($roleId, (string) $body['name']));
+
+        return new Response(StatusCodeInterface::STATUS_NO_CONTENT);
+    }
+
+    private function ensureContentTypeIsProvided(ServerRequestInterface $request): void
```

Comment:
I wonder if we would need a 415 ApiProblem, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/415

Reply:
It already exists in our list of error types: https://docs.publiq.be/docs/errors/baae4bf3cece5-header#unsupported-media-type

So should just be a matter of adding it to the `ApiProblem` class in this app